server/services/Mailer.js

```diff
@@ -3,24 +3,22 @@ const helper = sendgrid.mail;
 const keys = require('../config/keys');
 
 class Mailer extends helper.Mail {
-  constructor({ subject, recipients }, content) {
+  constructor(survey, content) {
     super();
 
     this.sgApi = sendgrid(keys.sendGridKey);
     this.from_email = new helper.Email('no-reply@emaily.com');
-    this.subject = subject;
+    this.subject = survey.subject;
     this.body = new helper.Content('text/html', content);
-    this.recipients = this.formatAddresses(recipients);
+    this.recipients = this.formatAddresses(survey.recipients);
 
     this.addContent(this.body);
     this.addClickTracking();
     this.addRecipients();
   }
 
   formatAddresses(recipients) {
-    return recipients.map(({ email }) => {
-      return new helper.Email(email);
-    });
+    return recipients.map(recipient => new helper.Email(recipient.email));
```

Comment:
Again a case of confusing parameter destructuring. Calling `.map` on an array takes a callback where a single item is the first argument. His code pulls out just the email as the parameter, but I find this very hard to ready.

I strongly pefer using a singular version of the array (a single `recipient` from a `recipients` array), and then using `recipient.email` instead of just email. 

Since this is a simple arrow function, I moved it to one line and removed the curly braces.

Again, does the same thing, but I find this much easier to understand.